### PR TITLE
vpnbypass: README update, code cleanup

### PR DESCRIPTION
**Pull request by stangri**
Maintainer: me
Compile tested: mvebu, WRT3200ACM, 19.07.3
Run tested: not tested

Description: updated README, minor code cleanup.

Please merge, I don't have merge rights to packages.

Signed-off-by: Stan Grishin <stangri@melmac.net>

### CODE REVIEW

**Comment by aparcar** [comment]
Do you mind adding a runtime test? It's documented [here](https://github.com/openwrt/packages/pull/13412/commits/def68fe80def02014e2ebfc4f84dbd9d3ba9b5c0) and an example is [here](https://github.com/openwrt/packages/blob/efe525033d078432c6031c533ece1a90c0916cc5/net/atftp/test.sh)

File: net/vpnbypass/Makefile [hunk 1]
```diff
@@ -5,7 +5,7 @@ include $(TOPDIR)/rules.mk
 
 PKG_NAME:=vpnbypass
 PKG_VERSION:=1.3.1
-PKG_RELEASE:=7
```
Comment (aparcar) [inline net/vpnbypass/Makefile:8]: Why do you jump from 7 to 9?

**Review by aparcar — CHANGES_REQUESTED** [review]

File: net/vpnbypass/Makefile (hunk 1 above)
Comment (stangri) [reply net/vpnbypass/Makefile:8]: I also offer most packages from private repo and bump release numbers there more frequently than I submit PRs for changes. In order not to have the lower version in the official repo, that means that sometimes there are jumps.

**Comment by stangri** [comment]
> Do you mind adding a runtime test? It's documented [here](https://github.com/openwrt/packages/pull/13412/commits/def68fe80def02014e2ebfc4f84dbd9d3ba9b5c0) and an example is [here](https://github.com/openwrt/packages/blob/efe525033d078432c6031c533ece1a90c0916cc5/net/atftp/test.sh)

Thank you very much, I appreciate the links! I'll push the update to this PR shortly.

File: net/vpnbypass/Makefile (hunk 1 above)
Comment (aparcar) [reply net/vpnbypass/Makefile:8]: Okay I guess.

**Review by aparcar — APPROVED** [review]

**Comment by stangri** [comment]
@aparcar should I force-push CI-related update to 19.07 and 18.06 branches?

**Comment by aparcar** [comment]
> @aparcar should I force-push CI-related update to 19.07 and 18.06 branches?

The CI currently lacks support for testing of older releases, however I'll add that later. Please keep them all in sync.

**Comment by aparcar** [comment]
I'm not really a fan of cosmetic backports but since you are the maintainer, your choice I guess.

**Pull request merged**